tests/storage-spec.js

```diff
@@ -155,7 +163,7 @@ describe('languageStorage factory', function () {
                     label: 'Polski',
                 },
                 {
-                    id: 'en_us',
+                    id: 'en',
```

Comment:
Is it not simple to have `languageIdMatch || stripCulture(languageIdMatch)` check?